lib/utils.c

```diff
@@ -39,6 +39,54 @@
 
 int timestamp_short;
 
+int read_prop(char *dev, char *prop, long *value)
+{
+	char fname[41], buf[80], *endp;
+	ssize_t len;
+	FILE *fp;
+	long result;
+
+	sprintf(fname, "/sys/class/net/%s/%s", dev, prop);
+	fp = fopen(fname, "r");
+	if (fp == NULL) {
+		if (strcasecmp(prop, "tun_flags"))
+			fprintf(stderr, "open %s: %s\n", fname,
+				strerror(errno));
```

Comment:
Once we delete the `if` statement that wraps this `fprintf()`, let's move it back on level of indentation. Also, make the error message read "fopen" instead of "open."